### PR TITLE
Generate room preview even when minimized

### PR DESCRIPTION
**Pull request by jaiwanth-v**
Fixes vector-im/element-web/issues/17056

ezgif com-gif-maker (13)
This doesn't affect the minimized UI.

Signed-off-by: Jaiwanth <jaiwanth2011@gmail.com>

### CODE REVIEW

**Comment by turt2live** [comment]
so the reason we did this was for performance reasons: if the list is minimized, we can save CPU by not generating the previews.

**Comment by jaiwanth-v** [comment]
I see. In that case, should the previews be generated on resize?

**Comment by turt2live** [comment]
possibly; not entirely sure. Hoping the reviewer has some ideas :D

**Review by jryans — CHANGES_REQUESTED** [review]
I agree with @turt2live that currently we don't do this while minimised to save on performance. Perhaps we can emit an event when un-minimising which triggers computation of all the previews?

**Comment by jaiwanth-v** [comment]
Yes, that would be a better idea. I'll implement that.

**Comment by turt2live** [comment]
(please avoid force pushes as it makes review 10x more difficult)

**Comment by jaiwanth-v** [comment]
Oh okay, sorry.

File: src/components/views/rooms/RoomTile.tsx [hunk 1]
```diff
@@ -102,6 +104,9 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
         };
         this.notificationState = RoomNotificationStateStore.instance.getRoomState(this.props.room);
         this.roomProps = EchoChamber.forRoom(this.props.room);
+        if (this.props.resizeNotifier) {
+            this.props.resizeNotifier.on("middlePanelResizedNoisy", this.onResize);
```
Comment (t3chguy) [inline src/components/views/rooms/RoomTile.tsx:108]: We should probably use `middlePanelResized` as we don't need sub-200ms responsiveness on this. Resizing fires a *LOT*

Comment (jaiwanth-v) [reply src/components/views/rooms/RoomTile.tsx:108]: Yes. I will change that.

**Review by jryans — APPROVED** [review]
Thanks, looks reasonable overall to me! 😄

**Pull request merged**